app/forms/state_file/id_disability_form.rb

```diff
@@ -7,6 +7,17 @@ class IdDisabilityForm < QuestionsForm
     validates :primary_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_primary_disabled? }
     validates :spouse_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_spouse_disabled? }
 
+    def initialize(intake, params = {})
+      super
+      already_answered_disability = !intake.primary_disabled_unfilled? && !intake.spouse_disabled_unfilled?
+      if intake.show_mfj_disability_options? && already_answered_disability && mfj_disability.nil?
+        previously_answered_mfj_disability = disabled_attrs_to_mfj_disability(
```

Comment:
`saved_mfj_disability_answer` or `previous_mfj_disability_answer` is probably the better name here

Reply:
I actually don't mind the name `previously_answered_mfj_disability`